snapmodel_policy.go

```diff
@@ -80,12 +57,8 @@ type SnapModelProfileParams struct {
 //  digest2 = H(digest1 || model)
 //  digestModel = H(digest2 || series || grade)
 // The signing key digest algorithm is encoded in little-endian format, and the sign-key-sha3-384 field is hashed in decoded (binary)
-// form. The brand-id, model and series fields are hashed without null terminators. The grade field is encoded as a single byte with
-// the following conversion:
-//  "unset":     0
-//  "secured":   1
-//  "signed":    2
-//  "dangerous": 3
+// form. The brand-id, model and series fields are hashed without null terminators. The grade field is encoded as the 32 bits from asserts.ModelGrade.Code.
```

Comment:
I would be explicit that the 4-byte grade field is encoded in little endian.

Reply:
done